packages/components/ContextualMenu/src/ContextualMenu.tsx

```diff
@@ -95,13 +103,7 @@ export const ContextualMenu = compose<ContextualMenuType>({
       <CMContext.Provider value={renderData.state.context}>
         <Slots.root>
           <Slots.container>
-            {Platform.OS === 'macos' ? (
-              <FocusZone focusZoneDirection={'vertical'}>{children}</FocusZone>
-            ) : (
-              <ScrollView contentContainerStyle={{ flexDirection: 'column', flexGrow: 1 }} showsVerticalScrollIndicator={true}>
-                {children}
-              </ScrollView>
-            )}
+            <Slots.scrollView>{Platform.OS === 'macos' ? <Slots.focusZone>{children}</Slots.focusZone> : { children }}</Slots.scrollView>
           </Slots.container>
```

Comment:
nit: I find this harder to parse/read than the format it was in before. Prettier probably won't let you split this into several lines?

Maybe assigning the internal part to a variable might help readability? ex

```tsx
const inner = Platform.OS === 'macos' ? <Slots.focusZone>{children}</Slots.focusZone> : { children };

...

<Slots.scrollView>{inner}</Slots.scrollView>
```

Reply:
Yeah, Prettier won't. I like that suggestion!

Reply:
I guess similar comment for Submenu

Reply:
Unfortunately, this optional rendering of a slot caused issues, presumable because `<Slots.focusZone>{children}</Slots.focusZone>` is of type `JSX.Element` while `{ children }` is of type `React.ReactNode[]`.

The E2E tests were failing with this error in the screenshot below. I was only able to fix it locally by just duplicating the entire tree inside an if/else 😢 

![error](https://user-images.githubusercontent.com/6722175/152604110-f966415a-05dc-4f69-b31e-c382357282d9.png)